packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -2315,6 +2305,7 @@
 	"settings.mfa.title": "Multi-factor Authentication",
 	"settings.mfa.updateConfiguration": "MFA configuration updated",
 	"settings.mfa.invalidAuthenticatorCode": "Invalid authenticator code",
+	"projects": "Projects",
```

Comment:
We should discuss with JC because in the end, there are projects but personal projects will be displayed as people, so we might want to name it "Source" or "Origin" rather than "Projects". Since it's just a UI thing, it shouldn't block merging.

Reply:
Ok, I like **Origin** but will wait for the final decision